src/Sulu/Bundle/AdminBundle/Navigation/NavigationItem.php

```diff
@@ -282,12 +288,28 @@ public function getDisabled()
     }
 
     /**
+     * @return null|string
+     */
+    public function getDataNavigationUrl()
+    {
+        return $this->dataNavigationUrl;
+    }
+
+    /**
+     * @param string $dataNavigationUrl
+     */
+    public function setDataNavigationUrl($dataNavigationUrl)
+    {
+        $this->dataNavigationUrl = $dataNavigationUrl;
+    }
+
+    /**
      * Returns a copy of this navigation item without its children
      * @return NavigationItem
      */
     public function copyChildless()
     {
-        $new = new NavigationItem($this->getName());
+        $new = $this->newInstance();
```

Comment:
If the reason you didn't use `clone` here are the children, then maybe you should call this method also `copyChildless`?